app/js/components/answers/Show.react.js

```diff
@@ -48,30 +54,45 @@ class AllAnswers extends React.Component {
 
    render () {
      let answer = this.props.answer || {};
+     let question = this.props.question || {};
      let user = answer.user || {};
      let current_user = AuthStore.getCurrentUser();
-     let answer_edit_btn, answer_delete_btn;
+     let answer_edit_btn, answer_delete_btn, answer_accept_btn, accepted_answer_ribbon, accepted_answer;
      let answer_date = new Date(answer.created_at);
      let answer_dom_id = `answer-${answer.id}`;
-     var text_to_copy = `http://${window.location.host + window.location.pathname}#${answer_dom_id}`;
+     
+     var answer_href = `http://${window.location.host + window.location.pathname}#${answer_dom_id}`;
```

Comment:
Lets be consistent with the use of var or let. let is preferable in this case because its a blocked scope variable.